tests/test_inspector.py

```diff
@@ -0,0 +1,85 @@
+"""Authors: Cody Baker and Ben Dichter."""
```

Comment:
ideally we would have unit tests, one for each check

Reply:
Yep, that's the idea of the `test_{neurodata_type}.py` files, which I've simplified a bit now. They test 3 levels; on the raw `neurodata_type` object that trigger the check, on the `neurodata_type` object contained within a `nwbfile` object, and the `neurodata_type` object contained within a written and read-in `nwbfile`.

For checks like `check_regular_timestamps`, I've generalized it so that its core logic depends on a `utils` function that works on any arbitrary arrays (not just time series). It's unit tests are then in `test_utils.py`. The idea here is for anything with really general application like that (including other functions from our `auto_qc.py` or `conversion_tools` on `nwb-conversion-tools`) can then be consolidated similarly for long-term reference across repositories.